TestEnvironmentSetup.md

```diff
@@ -0,0 +1,93 @@
+
+1. Create a MELD account
+    - Navigate to [Meld Sandbox](https://meld.interop.community)
+    - Select 'Register'
+    - Enter your information and select 'Register' again
+1. Create a MELD Sandbox
+
+    - Log into MELD
+
+    ![](./test-environment-images/meld_login.png)
+
+    - Select 'NEW SANDBOX'
+
+    ![](./test-environment-images/meld_sandboxes.png)
+
+    - Fill out the 'Create Sandbox' Form
+
+    ![](./test-environment-images/meld_create_sandbox.png)
+
+        - Give the Sandbox a Name and an Id (Id will be automatically generated)
+        - Select FHIR Version 'FHIR R4 (v4.0.1)
+        - Select all of the checkboxes
+            - Allow Open FHIR Endpoint
+            - Import sample patients and practitioners
+            - Import sample applications
+1. Import FHIR Resources from test-ehr repository
+    - Click settings in MELD in your desired sandbox and select 'Data Manager'
+    - Select the 'IMPORT' tab from the top then select 'LOAD FROM FILE'
+    ![](./test-environment-images/meld_import_resource.png)
+    - Navigate to the folder that the test-ehr is downloaded to and choose the fhirResourcesToLoad folder
+    - Individually select and upload each FHIR resource
+        - Note: you can skip the rems_smith* files if you only want the Jon Snow Patient
+    ![](./test-environment-images/meld_import_resource_done.png)
+1. Register the REMS SMART on FHIR application
+    - Navigate to the 'Apps' section of MELD
+    - Select the '+' button at the top right of the page
+    ![](./test-environment-images/meld_apps_1.png)
+    - Select 'Manually' for how to create the app
+    ![](./test-environment-images/meld_app_creation.png)
+    - Enter the following information into the App Details and leave the rest as default
+    ![](./test-environment-images/meld_smart_app_entry.png)
+        - App Name: REMS Smart App (Local)
+        - App Launch URI: http://localhost:4040/launch
+        - App Redirect URIs: http://localhost:4040/index
+        - Scopes: patient/Patient.read openid profile user/Patient.read offline_access launch user/Practitioner.read
+    - Select save and store the Client ID that is generated and displayed in a pop-up, this will be needed later
+    ![](./test-environment-images/meld_smart_app_registered.png)
+1. Register the Request Generator
+    - Complete the steps from the previous section with the following information for the request generator instead
+    ![](./test-environment-images/meld_request_generator_entry.png)
+        - App Name: Request Generator (Local)
+        - App Launch URI: http://localhost:3000/launch
+        - App Redirect URIs: http://localhost:3000/#/index, http://localhost:4040/register,http://localhost:3000/index, http://localhost:4040/index
+        - Scopes: launch openid user/*.* offline_access profile
```

Comment:
The *.* isn't showing up in markdown - update to show *.*

Reply:
fixed